### DISCUSSION TITLE
Octopus Intelligent Node Red

### DISCUSSION THREAD

**Discussion by joelyroley** [BottlecapDave/HomeAssistant-OctopusEnergy#1061] [Q&A]
Hello,

Would it be possible to use API information you obtain for intelligent go dispatch to stop my victron inverter from powering my house during the cheap period in Node Red

**Comment by BottlecapDave** [comment]
Hello. I'm not very familiar with node red, but you should be able to pull the data from the attributes from the dispatch sensor (assuming you're not using the service via ohme). You might have better luck on the official HA forums stating the attributes that are available on the sensor that you want to extract.
